src/assets/dotnet.md

```diff
@@ -1,5 +1,66 @@
 ## .NET
 ---
 ### Prerequisites
+To use AllureNUnit in a C# project it is necessary install:
+- Download Nuget NUnit 3 https://nunit.org/
+- Download Nuget NUnit.Allure v.1.0.6-beta30 https://www.nuget.org/packages/NUnit.Allure
+- Configure allureConfig.json
+- Set AllureNUnit attribute under test fixture
+- Use other attributes if needed 
 
 ### Writing a Test case
+The [AllureNUnit] tag it is necessary to generate the JSON files.
+The [Description] tag is the step test case.   
+The [AllureTms] tag is the test case name defined in Jama.
+
+NUnit.Allure attribute | Allure model location | allure-reporter field | JAMA field
+--- | --- | --- | --- 
+AllureEpic | as `label` with name `epic` | Not supported | Not supported
+AllureTms | as `links` field `name` | Title | Test Name
+AllureFeature | `feature` | Subtitle | Not supported
+Description | `action` | Action | Action
+
```

Comment:
Please add a note explaining how your Expepected Results are managed by JSON, see that the tag [step] is not implemented yet.